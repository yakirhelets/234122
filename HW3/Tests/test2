getApartments sourceOne "J. Cole" 5
