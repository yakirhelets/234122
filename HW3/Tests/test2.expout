12 2000 050-0000111
11 3000 050-0000111
