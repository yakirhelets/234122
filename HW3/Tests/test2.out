11 3000 050-0000111
12 2000 050-0000111
